ci/provisioning/post_provision_config.sh

```diff
@@ -12,13 +12,53 @@ host wolf-*
     LogLevel error
 EOF
 
+: "${DAOS_STACK_RETRY_DELAY_SECONDS:=60}"
+: "${DAOS_STACK_RETRY_COUNT:=3}"
+: "${BUILD_URL:=Not_in_jenkins}"
+: "${OPERATIONS_EMAIL:=$USER@localhost}"
+
+retry_cmd() {
+    local command="$1"
+    shift
```

Comment:
Do we need to pull off the first part of the command string?  Can't we just use `$@` on lines 25 & 33?